src/Bindings/ManualBindings.cpp

```diff
@@ -2895,7 +2887,74 @@ class cLuaBlockTracerCallbacks :
 
 protected:
 	cLuaState::cTableRefPtr m_Callbacks;
-} ;
+};
+
+
+
+
+
+/** Provides interface between a Lua table of callbacks and the cBlockTracer::cCallbacks */
```

Comment:
Clarify in the doxy-comment why there are two such classes and why this one's `Old`

Reply:
It should be better now.